build.sbt

```diff
@@ -539,6 +540,21 @@ lazy val `kamon-caffeine` = (project in file("instrumentation/kamon-caffeine"))
     )
   ).dependsOn(`kamon-core`, `kamon-testkit` % "test")
 
+lazy val `kamon-finagle` = (project in file("instrumentation/kamon-finagle"))
```

Comment:
Please run `headerCreate` on the project to get all the license-related headers.